### PR TITLE
Static web app

### PR DESCRIPTION
**Pull request by JackMeadDev**
Add in example code & pipeline for deploying this site as a static web app. Demo deployment has run, but not DNS changes applied:

Site can be viewed here:
https://black-mushroom-035508903.4.azurestaticapps.net/

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          run: terraform init
+          working-directory: infrastructure
+        - name: Create the infrastructure
+          run: terraform apply -auto-approve
+          working-directory: infrastructure
+
+        - name: Install static web app CLI
+          run: npm install -g @azure/static-web-apps-cli
+        - name: Install credential helper
+          run: sudo apt install gnome-keyring pass
```
Comment (Jonesey13) [inline .github/workflows/cd.yml:31]: Assuming this is needed for the `swa deploy` command below?

**Review by Jonesey13 — APPROVED** [review]
Looks all sensible to me :)

**Comment by JackMead** [comment]
Closing in favour of:
https://github.com/corndeladmin/static-web-app-auth/

File: .github/workflows/cd.yml (hunk 1 above)
Comment (JackMead) [reply .github/workflows/cd.yml:31]: It was a suggested solution, I think I eventually determined not required, but not following up on this repo so will leave here

**Pull request closed without merging**